libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -178,20 +169,10 @@ class RouterWrapperTests {
         every {
             NativeRouteParserWrapper.parseDirectionsResponse(any(), any(), any())
         } answers {
-            val routesCount =
-                JSONObject(this.firstArg<String>())
-                    .getJSONArray("routes")
-                    .length()
-            val nativeRoutes = mutableListOf<RouteInterface>().apply {
-                repeat(routesCount) {
-                    add(
-                        mockk {
-                            every { routeId } returns "$it"
-                            every { routerOrigin } returns com.mapbox.navigator.RouterOrigin.ONBOARD
-                        }
-                    )
-                }
-            }
+            val nativeRoutes = createRouteInterfacesFromDirectionRequestResponse(
+                response = firstArg(),
+                requestUri = secondArg(),
+            )
```

Comment:
@dzinad , this is a case I often have with mocked DTO. I just added more logs and tests started to fail because the field I log isn't configured. Never happens with real DTO.
I also find a real DTO a bit easier to create when you already have infrostruacte from other tests which use the same DTO.

Reply:
> I just added more logs and tests started to fail because the field I log isn't configured.

I use `relaxed = true`. I'm actually not sure why this behaviour default with mockk. There was no such problem with Mockito.